content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,137 @@
+
+The party planner generates the condition and fulfillment values. The party planner provides the condition value to the party host, who creates the escrow. The part planner also provides the condition to the party band so that they know that this is the right condition.
+
+The party planner must keep the fulfillment value a secret. Anyone can use the condition and fulfillment values to finish the escrow, but in this scenario, it will most likely be the party band who will finish the escrow because they're the ones who are motivated to get paid.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Calculate time values needed for the escrow](send-a-conditionally-held-escrow.html#2-calculate-release-or-cancel-time) (sender)
+
+Because the participants want the escrow to be eligible for cancellation after 12 noon the day after the party, the party host (sender) must calculate a `CancelAfter` value to include in the escrow definition.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Create the escrow](send-a-conditionally-held-escrow.html#3-submit-escrowcreate-transaction) (sender)
+
+The party host (sender) creates the escrow that provides the smart contract. The party host must create the escrow because they are the only participant that can authorize the lock up and potential payout of XRP from their XRP Ledger account.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Wait for validation](send-a-conditionally-held-escrow.html#4-wait-for-validation) and [confirm escrow creation](send-a-conditionally-held-escrow.html#5-confirm-that-the-escrow-was-created) (sender and receiver)
+
+The party host (sender) waits for validation of the ledger that contains the escrow creation transaction and then confirms that the escrow was created.
+
+The party host then provides the escrow transaction's `hash` value to the party band (receiver). The party band can use the `hash` value to look up the escrow transaction on the XRP Ledger to ensure that it was created according to the smart contract terms they agreed to. As part of this step, the party band should also confirm that the condition matches the one the party planner (oracle) provided. If the condition's wrong, the fulfillment the party planner provides won't let the party band finish the escrow and get paid.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Finish the escrow](send-a-conditionally-held-escrow.html#6-submit-escrowfinish-transaction) (receiver)
+
+The party band (receiver) shows up and plays their set.
+
+The party planner (oracle) confirms that the party band has fulfilled their contract and publishes the fulfillment publicly, or at least to the party band.
+
+The party band finishes the escrow using the fulfillment value published by the party planner. The party band must finish the escrow before 12 noon or the escrow expires, the party band doesn't get paid, and the XRP returns to the party host's (sender's) XRP Ledger account.
```

Comment:
> The party band must finish the escrow before 12 noon or the escrow expires, the party band doesn't get paid, and the XRP returns to the party host's (sender's) XRP Ledger account.

I suggest breaking this up into two sentences so that it doesn't read as a list. Also, the next paragraph explains how the XRP doesn't automatically return, so it's probably less confusing if you just leave that part off.

> The party band must finish the escrow before 12 noon. If they don't, the escrow expires and the party band doesn't get paid.